sycl/cmake/modules/AddSYCL.cmake

```diff
@@ -50,7 +50,8 @@ function(add_sycl_plugin PLUGIN_NAME)
       ${sycl_inc_dir}
     LIBRARIES
       ${ARG_LIBRARIES}
-      OpenCL-Headers
+      OpenCL-Headers # TODO: remove dependency on OpenCL
+      UnifiedRuntime-Headers
```

Comment:
`UnifiedRuntime-Headers` seems to be added by sycl/plugins/unified_runtime/CMakeLists.txt, right? If I am reading sycl/plugins/CMakeLists.txt correctly, the `unified_runtime` plugin sub-directory is only added if the `level_zero` plugin in enabled, i.e.
```cmake
# level_zero plugin depends today on unified_runtime plugin
# and unified_runtime plugin is not an independent plugin, adding it explicitly
if ("level_zero" IN_LIST SYCL_ENABLE_PLUGINS)
  add_subdirectory(unified_runtime)
endif()
```
Since a lot more projects would depend on the UR headers after this, should we consider either enabling the `unified_runtime` plugin in more cases or move the logic for getting the headers?

Reply:
Good point! Yes, I made UR be added always now that all plugins depend on it (through PI->UR dependency).
When entire UR is moved into its separate repo we should consider greater restructure, like moving UR one level up.